full-service/src/db/account.rs

```diff
@@ -683,13 +715,13 @@ mod tests {
             id: 1,
             account_id_hex: account_id_hex.to_string(),
             account_key: [
-                10, 34, 10, 32, 129, 223, 141, 215, 200, 104, 120, 117, 123, 154, 151, 210, 253,
-                23, 148, 151, 2, 18, 182, 100, 83, 138, 144, 99, 225, 74, 214, 14, 175, 68, 167, 4,
-                18, 34, 10, 32, 24, 98, 18, 92, 9, 50, 142, 184, 114, 99, 34, 125, 211, 54, 146,
-                33, 98, 71, 179, 56, 136, 67, 98, 97, 230, 228, 31, 194, 119, 169, 189, 8, 26, 17,
-                102, 111, 103, 47, 47, 115, 111, 109, 101, 46, 102, 111, 103, 46, 117, 114, 108,
-                42, 23, 13, 231, 226, 158, 43, 94, 151, 32, 17, 121, 169, 69, 56, 96, 46, 182, 26,
-                43, 138, 220, 146, 60, 162,
+                10, 34, 10, 32, 135, 37, 92, 17, 29, 203, 205, 96, 6, 72, 229, 193, 11, 153, 50, 3,
```

Comment:
Curious the reason why this changed, given the rootID is from the same seed?

Reply:
my understanding is that it's due to the use of rng to generate the mock ledger db that was added earlier in the test, effectively changing the resulting root id

Reply:
Got it, ty!